src/send-pfring.h

```diff
@@ -16,9 +16,9 @@
 #error "Don't include send-bsd.h or send-linux.h with send-pfring.h"
 #endif
 
-int send_run_init(sock_t socket)
+int send_run_init(anonymous struct socket, int i, struct shard *pShard)
```

Comment:
The anonymous struct here worries me a little bit versus having two explicitly different functions that encapsulate the different behavior.

Reply:
I'm honestly not sure why I made this change. I didn't touch anything related to `pfring`, and all the other `send_run_init`'s have the old format. So I'll revert this change.